sentry/src/main/java/io/sentry/NoOpSpan.java

```diff
@@ -0,0 +1,50 @@
+package io.sentry;
+
+import io.sentry.protocol.SentryId;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class NoOpSpan implements ISpan {
+  @Override
+  public @NotNull ISpan startChild() {
+    return new NoOpSpan();
+  }
+
+  @Override
+  public @NotNull ISpan startChild(@Nullable String operation, @Nullable String description) {
+    return new NoOpSpan();
+  }
+
+  @Override
+  public @NotNull SentryTraceHeader toSentryTrace() {
+    return new SentryTraceHeader(SentryId.EMPTY_ID, SpanId.EMPTY_ID, false);
+  }
+
+  @Override
+  public void finish() {}
+
+  @Override
+  public void setOperation(@Nullable String operation) {}
+
+  @Override
+  public void setDescription(@Nullable String description) {}
+
+  @Override
+  public void setStatus(@Nullable SpanStatus status) {}
+
+  @Override
+  public void setThrowable(@Nullable Throwable throwable) {}
+
+  @Override
+  public @Nullable Throwable getThrowable() {
+    return null;
+  }
+
+  @Override
+  public @NotNull SpanContext getSpanContext() {
+    return new SpanContext();
```

Comment:
better creating a `SpanContext` using `SentryId.EMPTY_ID` where possible, also `sampled=false` like you do with `SentryTraceHeader`

Reply:
👍